draft-ietf-quic-transport.md

```diff
@@ -1076,8 +1076,8 @@ A zero-length connection ID can be used when a connection ID is not needed to
 route to the correct endpoint. However, multiplexing connections on the same
 local IP address and port while using zero-length connection IDs will cause
 failures in the presence of peer connection migration, NAT rebinding, and client
-port reuse; and therefore MUST NOT be done unless an endpoint is certain that
-those protocol features are not in use.
+port reuse. An endpoint MUST NOT use an IP address and port pair for multiple
+connections unless it is certain that those protocol features are not in use.
```

Comment:
This misses the zero-length connection ID qualification, which is the key point.

Reply:
Tried to address this.